src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupProjectsTable.jsx

```diff
@@ -11,7 +12,14 @@ import { setBaseUrl } from "../../../utilities/url-utilities";
  * @constructor
  */
 export function UserGroupProjectsTable({ groupId }) {
-  const [projects, setProjects] = useState();
+  const [projects, setProjects] = React.useState();
+  const [total, setTotal] = React.useState(0);
+  const [paginationOptions, setPaginationOptions] = React.useState(null);
+
+  React.useMemo(() => {
```

Comment:
This should be:

```jsx
const paginationOptions = React.useMemo(() => {
  getPaginationOptions(total);
}, [total]);
```

Reply:
Updated in c96bd77d27718c9064e6535c7aac61dd11b3f80b